contracts/interfaces/0.8.x/IMintingAPICoreV0.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IEngineContractMintingAPIV0 {
+    /**
+     * @notice Token ID `_tokenId` minted on project ID `_projectId` to `_to`.
+     */
+    event Mint(
```

Comment:
Just wanted to confirm explicitly, the minting API uses the `Mint` event? 🙏

Reply:
Yup, it does 👍